src/services/user_service.rs

```diff
@@ -26,6 +27,14 @@ pub struct UserResponse {
     pub first_login: bool,
 }
 
+#[derive(Debug, Clone, Deserialize)]
+pub struct CreateUserRequest {
+    pub email: Email,
+    pub username: String,
+    pub password: Password,
+    pub groups: Vec<String>,
```

Comment:
Is it a good idea to add the first login option here, for the user to change their password on their first login?